ui/src/app/views/bug-report/component.css

```diff
@@ -0,0 +1,13 @@
+.bug-report {
+  position: fixed;
+  bottom: 0px;
+  right: 0px;
+}
+
+.input {
+  width: 100%;
```

Comment:
This can sometimes lead to unintentional horizontal scrolling. You might combine this with `box-sizing: border-box` or (more reliable in my experience), using flex box.

Reply:
Testing this, I see `border-box` is part of Clarity's root styles. That's a little scary, but I'm open to seeing how it goes.

Reply:
Cool! It is a bit unnerving, but if it comes in as a root style from the start, it sounds like a good thing to me.